index.bs

```diff
@@ -306,14 +306,38 @@ partial interface XRSession {
   readonly attribute XRDepthUsage depthUsage;
   readonly attribute XRDepthDataFormat depthDataFormat;
   readonly attribute XRDepthType? depthType;
+  readonly attribute boolean? depthActive;
+
+  undefined startDepth();
+  undefined stopDepth();
```

Comment:
Feel free to ignore, but after reading through this I think I have a _very_ mild preference for naming these `pauseDepth()` and `resumeDepth()`, because requesting the session with depth sensing enabled is the "start" and we don't want to imply that you can arbitrarily start depth sensing at any point after the session has been started.

But that's a pretty obscure complaint. :grin: I take no offense if `start` and `stop` sits better with you!